exo-setup/src/service-setup.ls

```diff
@@ -9,6 +9,7 @@ require! {
 }
 
 
+# Runs build command in each of the services
 class ServiceSetup extends EventEmitter
```

Comment:
The setup commands that @kevgo is talking about can be defined in `service.yml`, and then ran when the image is built. It can be part of our Dockerfile template to just take this command from `service.yml`

Whenever `package.json` or `service.yml` is modified then `exo setup` has to be re-run so the Docker image can be rebuilt. Otherwise build and dependency commands can be run during the build phase of a Docker image and this file can be removed.

I've made a ticket to track this: https://github.com/Originate/exosphere-sdk/issues/283